src/utils/deploy.ts

```diff
@@ -126,18 +135,9 @@ export class DeployUtils {
       await DVNUtils.configureDVN(0);
 
       // ask user to setup DVNs on-chain
-      const setupDVNs = await select({
-        message: 'Do you want to setup DVNs on-chain?',
-        choices: [
-          {
-            name: 'Yes',
-            value: true,
-          },
-          {
-            name: 'No',
-            value: false,
-          },
-        ],
+      const setupDVNs = await confirm({
+        message: 'Do you want to write DVNs on-chain?',
```

Comment:
Curious, what does it actually mean `to write DVNs on -chain` ?

Reply:
had to write Wire instead of write

Reply:
oh ok, lets change it then when u can